packages/react-components/components/PhoneNumberInput.test.tsx

```diff
@@ -97,4 +104,30 @@ describe('PhoneNumberInput', () => {
       )
     })
   })
+
+  describe('Native phone picker', () => {
+    it('can read phone', async () => {
+      const wrapper = shallow<PhoneNumberInput>(
```

Comment:
This is okay, but FYI we prefer to use `react-native-testing-library` instead of enzyme for these kinds of tests.
Both are okay but we've found RNTL is easier to use :)